share/trick/makefiles/trickify.mk

```diff
@@ -92,62 +111,46 @@ TRICK_CXXFLAGS += $(TRICKIFY_CXX_FLAGS)
 # Ensure we can process all headers
 TRICK_EXT_LIB_DIRS :=
 
-# Trick library build type, defaulted to PLO
-# PLO - Partially-linked object library
-# STATIC - Static library
-# SHARED - Dynamic shared library
-TRICK_LIB_BUILD_TYPE ?= PLO
-
-# Add project source objects to trickified library.
-SRC_OBJS ?=
-
-# When given a library, the linker will only link in objects that are known to
-# be needed at link time. However, Trick uses dlsym to dynamically load the
-# objects we'll be creating here. It cannot be known which objects will be
-# needed at link time, so the sim has to link them all. In that case, we might
-# as well create an object (which will be fully linked in by the sim) instead
-# of a library (which would require the use of the -whole-archive option).
-#
-# One disadvantage of this approach is that we have to link all of the objects
-# in this rule no matter how many have changed, whereas ar would allow us to
-# replace only the changed objects. However, Trickified projects are
-# necessarily used as third-party libraries, and so would be expected to
-# change infrequently. Moreover, the methods for force-linking a library differ
-# between Linux and Mac, so obviating the need for it simplifies a Trickified
-# project's user-facing makefile.
-$(TRICKIFY_OBJECT_NAME): $(SWIG_OBJECTS) $(IO_OBJECTS) $(SRC_OBJS) | $(dir $(TRICKIFY_OBJECT_NAME))
+$(TRICKIFY_OBJECT_NAME): $(SWIG_OBJECTS) $(IO_OBJECTS) | $(dir $(TRICKIFY_OBJECT_NAME))
```

Comment:
I see the option to include the source code $SRC_OBJS  in to the trickified library will be left out. Is that right? I only tested it with SHARED, not sure if it works for PLO or STATIC, might be the reason.

Reply:
That's correct. See my [comment](https://github.com/nasa/trick/pull/743#issuecomment-476271437) above.